### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by yalh76**
## Problem

- *Level 0*

## Solution

- *Upgrade to 2022.08.30~ynh1*
- *Apply last example_ynh*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Review by FabianWilkens — APPROVED** [review]
Thank you for your work.

I now see what the problem was.
It probably would have taken me a while to figure that out.

File: conf/systemd_api-gateway.service [hunk 1]
```diff
@@ -10,6 +10,7 @@ User=__APP__
 Group=__APP__
 WorkingDirectory=__FINALPATH__/live/
 EnvironmentFile=__FINALPATH__/live/api-gateway.env
+Environment="__YNH_NODE_LOAD_PATH__"
```
[the remark this answers is not in the document]
Comment (yalh76) [reply conf/systemd_api-gateway.service]: removed from all services

File: scripts/upgrade [hunk 2]
```diff
@@ -202,125 +195,123 @@ syncing_server_version_installed=$(ynh_app_setting_get --app=$app --key=syncing_
...
 
-    chmod -R 750 "$datadir"
-    chmod -R o-rwx "$datadir"
-    chown -R $app:$app "$datadir"
+	chmod -R 750 "$datadir"
+	chmod -R o-rwx "$datadir"
+	chown -R $app:$app "$datadir"
 fi
 
+ynh_app_setting_set --app=$app --key=standalone --value="true"
```
[the remark this answers is not in the document]
Comment (yalh76) [reply scripts/upgrade]: removed

**Pull request merged**